lib/explorer/shared.ex

```diff
@@ -155,7 +160,7 @@ defmodule Explorer.Shared do
   def cast_numerics(list, type) when type == :numeric do
     data =
       Enum.map(list, fn
-        nil -> nil
+        item when is_atom(item) -> item
```

Comment:
I think we could have an "allow list" of the values we accept here.
Something like:

```suggestion
        item when item in [nil, :infinity, :neg_infinity, :nan] -> item
```

WDYT?

Reply:
Yes, I agree.